src/main/java/uk/gov/hmcts/reform/sendletter/services/date/DateService.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.sendletter.services.date;
+
+import org.springframework.stereotype.Service;
+
+import java.time.ZonedDateTime;
+
+@Service
+public class DateService {
+
+    public ZonedDateTime subtractBusinessDays(ZonedDateTime dateTime, int numberOfBusinessDays) {
+        int daysSubtracted = 0;
+        ZonedDateTime adjustedDateTime = dateTime;
+
+        while (daysSubtracted < numberOfBusinessDays) {
```

Comment:
We could calculate the day instead of iterating, but when we start taking bank holidays into account, that approach would be more complicated and given that we're only going to subtract a couple days, not worth it.